manager/manager.go

```diff
@@ -144,8 +144,9 @@ func (rm *realManager) HousekeepModel() {
 
 func (rm *realManager) Housekeep() {
 	for {
-		start := rm.lastSync
-		end := start.Add(rm.resolution)
+		end := rm.lastSync.Add(rm.resolution)
+		// We need only one sample every housekeeping. Issue #638.
+		start := end.Add(-time.Second)
```

Comment:
I also think the right solution here will be for cAdvisor or kubelet to support resolution.